nutaudet/NuTauMudet.cxx

```diff
@@ -511,8 +508,7 @@ void NuTauMudet::ConstructGeometry()
       TGeoVolume *volIron = new TGeoVolume("volIron",IronLayer,Iron);
       volIron->SetLineColor(kGray);
 
-      TGeoBBox *IronLayer1 = new TGeoBBox("Iron",fXFe/2, fYFe/2, fZFethin/2);
-      TGeoVolume *volIron1 = new TGeoVolume("volIron1",IronLayer1,Iron);
+      auto* IronLayer1 = new TGeoBBox("Iron",fXFe/2, fYFe/2, fZFethin/2);
```

Comment:
Is this used anywhere?

Reply:
Turns out it is used in line 573. I would suggest skipping the rename on line 551, which makes it harder to find that use and give it the name you want immediately.
Same for IronLayer.

Reply:
Thank you, I have moved the SetName() lines above.

Reply:
Thanks, but what I meant is, why don't you just `new TGeoBBox("MUDETIRON"...)`? You can just set the name in the constructor.

Reply:
This is now the only outstanding issue. Otherwise things look good to be merged.

Reply:
Ah, I understand now, sorry!